core/src/main/java/nl/nn/adapterframework/util/CompactSaxHandler.java

```diff
@@ -165,13 +184,13 @@ private String elementsToString() {
 		return chain;
 	}
 
-	public String getXmlString() {
-		return messageBuilder.toString();
+	@Override
+	public String toString() {
+		return getContentHandler().toString();
 	}
```

Comment:
Dit is niet zo handig denk ik?

Reply:
Ik had dat van `PrettyPrintFilter` overgenomen en dacht dat jullie dat een goed idee vonden. Ik maak er wel weer wat anders van.